public/views/partials/lib/hooks/hook_headscripts.liquid

```diff
@@ -9,6 +9,14 @@
     {%- for color in theme_variables.colors -%}
       --{{color.name}}: {{color.value}};
     {%- endfor -%}
+    {% comment %} border radius variables {% endcomment %}
```

Comment:
Would you mind adding the default values to the default theme config?
https://github.com/Platform-OS/pos-module-theme-manager/blob/master/public/modules/default-theme/public/views/partials/lib/hooks/hook_theme_variables.liquid

I know that it doesn't make any difference on the frontend but it'd make it more clear for developers.